README.md

```diff
@@ -40,30 +40,21 @@ If you are building for a local development network, ganache has to be running l
 yarn run ganache # start a development network (blocking)
 ```
 
-In case only the StableX or only the SnappAuction should be deployed, you can add the flags `--onlyMigrateStableX` or `--onlyMigrateSnappAuction` to the deployment command.
-
-E.g:
-
-```
- npx truffle migrate --onlyMigrateStableX=true
-```
-
 3. Verify the contracts for some cool Etherscan.io goodies (see below for more help)
 
 ```sh
-npx truffle run verify SnappAuction --network $NETWORKNAME
 npx truffle run verify BatchExchange --network $NETWORKNAME
 ```
 
 4. List some default tokens on the StableX exchange
```

Comment:
As a separate PR should we clean up the "StableX" references and replace them with "Gnosis Protocol"?